src/agent/state.rs

```diff
@@ -1,7 +1,7 @@
-pub const STORAGE_KEY_CSRF_TOKEN: &str = "ctron/oauth2/csrfToken";
-pub const STORAGE_KEY_LOGIN_STATE: &str = "ctron/oauth2/loginState";
-pub const STORAGE_KEY_REDIRECT_URL: &str = "ctron/oauth2/redirectUrl";
-pub const STORAGE_KEY_POST_LOGIN_URL: &str = "ctron/oauth2/postLoginUrl";
```

Comment:
If we are going to make this specific const private, it might be a good idea to expose some method to get it, if user wants to for example provide a "redirect back" link from landing page.

Reply:
I was trying to hide that internal state as far away as possible … would it work better to have some struct/fn which returns the actual data?

Reply:
I exposed this as the `LoginState`.

Reply:
That looks amazing :)